src/main/java/edu/ie3/datamodel/io/source/ThermalSource.java

```diff
@@ -121,7 +125,8 @@ public Map<UUID, ThermalBusInput> getThermalBuses(Map<UUID, OperatorInput> opera
    * @return a map of UUID to object- and uuid-unique {@link ThermalStorageInput} entities
    */
   public Map<UUID, ThermalStorageInput> getThermalStorages() throws SourceException {
-    return getCylindricalStorages().stream().collect(toMap());
+    return Stream.concat(getCylindricalStorages().stream(), getDomesticHotWaterStorages().stream())
+        .collect(Collectors.toMap(ThermalStorageInput::getUuid, storage -> storage));
```

Comment:
Why did you change the collector?

Reply:
I mean, why don't you use `collect(toMap())`?

Reply:
Good question, will adapt it.